site/src/pages/community/articles.mdx

```diff
@@ -342,6 +342,14 @@ Send a pull request by editing [this page](https://github.com/line/armeria/edit/
   type="video"
 />
 
+<Article
+  title="비동기 서버 그까이꺼, Request Scoping만 알면 끝!"
```

Comment:
Request scoping is so important - hoping to see @minwoox's English dub soon :)

Reply:
🤣